src/store/modules/wallets/wallets-plugin.ts

```diff
@@ -0,0 +1,14 @@
+import VuexPersistence from 'vuex-persist'
+import { CoinSymbol, PluginWalletsState } from '@/store/modules/wallets/types'
+
+const walletsPersistencePlugin = new VuexPersistence({
+  key: 'adm-wallets',
+  storage: window.localStorage,
+  reducer: (state: PluginWalletsState) => {
+    return {
+      symbols: state.wallets.symbols as CoinSymbol[]
```

Comment:
```suggestion
      symbols: state.wallets.symbols
```

I guess it will work